docs/Auth-with-Ionic2.md

```diff
@@ -467,51 +466,49 @@ and update the signInWithFacebook() method.
 your ```auth-service.ts``` code should look like this.
 
 ```ts
-
-
+import { Observable } from 'rxjs/Observable';
 import { Injectable } from '@angular/core';
-import { AuthProviders, AngularFireAuth, FirebaseAuthState, AuthMethods } from 'angularfire2';
+import { AngularFireAuth } from 'angularfire2/auth';
+import * as firebase from 'firebase/app';
```

Comment:
Make a comment here about how this import is important because it doesn't pull in the entire SDK whereas `import * as firebase from 'firebase';` will.